pycoal/mineral.py

```diff
@@ -2,132 +2,219 @@
 
 import spectral
 import pickle
+import numpy
 
 # define number of layers in normalized pixel
 NUMBER_OF_LAYERS = 128
 
-# define wavelength range
-WAVELENGTH_MIN = ???
-WAVELENGTH_MAX = ???
+# define wavelength range in micrometers
+WAVELENGTH_MIN = 0.38315
+WAVELENGTH_MAX = 2.5082
+
+class MineralClassification:
+    def __init__(self):
+        # maybe we could put the constants above in here?
+        # any other variables to move in here?
```

Comment:
i should add that i expect these to be immutable values. i believe the spectral library and the images must be normalized to the same dimensions for training and classification to work.

there is a potential use case for letting the application programmer set these, but only if they are training their own network from scratch. if they reuse a trained classifier we redistribute, the value must be the same as what we initially trained it with.

i can only speak for the neural net implementation, but i would imagine that this requirement would be the same for other kinds of classifiers. the goal being to make the training data and the data to classify as uniform, and as small, as possible.